### DISCUSSION TITLE
Can I show always y-axis maximum?

### DISCUSSION THREAD

**Discussion by sakai-IST** [oxyplot/oxyplot#1961] [Q&A] [answered]
![キャプチャ](https://user-images.githubusercontent.com/98733014/203251241-4bbc62c3-2546-4bbf-addb-0bd10efb6130.PNG)

Is it possible to always display the maximum value of the Y-axis of a graph?
Sometimes the maximum value is not displayed, as shown in this picture.

**Comment by VisualMelon** [comment, marked as answer]
You can force it by overriding `GetTickValues` in the axis, but not without extending the appropriate axis class. You can encourage it by reducing the `IntervalLength` property for the axis (i.e. reduce the minimum distance between ticks: default is 60 screen-space units) e.g. if you know your axis will always be small.

Linking https://github.com/oxyplot/oxyplot/pull/1712 which aims to address this sort of issue, and I should probably get on and merge.

**Reply by sakai-IST** [reply]
Thanks for your reply and sorry my late reply.
`IntervalLength` property is worked!
